client/components/input/input.html

```diff
@@ -1,3 +1,95 @@
-<link rel="stylesheet" href="./input.css">
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+  <meta charset="UTF-8">
+  <title>Input</title>
+  <link rel="stylesheet" href="./input.css">
+  
 
-<input class="unique-input-class" type="text" />
+</head>
+<body>
+
+<div id ="inputText" class= "fieldset" > 
+  <fieldset id= "fieldName">      
+             <input 
+                    class="input heading-h5"
+                    id = "text1" 
+                    type="text" 
+                    name="textBox"  
+                    minlength="3" 
+                    maxlength="25"
+                    placeholder="Text"
+                    size="25"
+                    pattern="[a-z]{1,15}" 
+                    required>
+                    <b></b>              
+  <label class="input-label" for="text1">What to refund:</label>
+  </fieldset>
+</div>
+
+<div id ="inputNumber" class= "numberInput" > 
+   <fieldset id= "fieldNumber">
+                <input 
+                    class="input heading-h5"
+                     id = "number1"   
+                     type="number" 
+                     name="nameOfInput" 
+                     min="0" 
+                     max="99000" 
+                     value=""
+                     step=".01"
+                     placeholder="Number"
+                     required  >
+                  <label class="input-label" for="number1">Amount:</label>
+     </fieldset>
+ </div>
+  
+ <div id ="inputText" class= "fieldset" > 
+  <fieldset id= "fieldName">      
+             <input class="inputNoLabel heading-h5"
+                    id = "text1" 
+                    type="text" 
+                    name="textBox"  
+                    minlength="0" 
+                    maxlength="25"
+                    placeholder="Text"
+                    size="25" 
+                    required>               
+  </fieldset>
+</div>
+
+<div id ="inputText" class= "fieldset" > 
+  <fieldset id= "fieldName">       
+             <input 
+                    class="input heading-h5"
+                    id = "text1" 
```

Comment:
You are using `text1` id 3 times on same html page. You shouldn't do that. Id has to be always unique per page.
Also it's better give clear name to identifier. In this case for example could be `what-to-refund`